manifests/repo.pp

```diff
@@ -0,0 +1,17 @@
+class katello::repo (
+  $manage_repo = $::katello::manage_repo,
+  $repo        = $::katello::repo,
+  $dist        = $::katello::repo_yumcode,
+  $gpgcheck    = $::katello::repo_gpgcheck,
+  $gpgkey      = $::katello::repo_gpgkey,
+) {
+  if $manage_repo {
```

Comment:
Oh I think I forgot the include there. The reason I put it here is that this makes chaining easy